specs/CalendarView/CalendarViewSpec1.md

```diff
@@ -0,0 +1,498 @@
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+A new set of CalendarView styling properties are introduced to allow greater customization of the control's rendering in upcoming Windows OS releases.
+These properties will be used in WinUI to align the CalendarView's styling with the rest of the standard controls.
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+## Today styling
+The following properties of type Windows.UI.Xaml.Media.Brush are used to customize the current day's rendering when CalendarView.IsTodayHighlighted is True:
+TodayBackground, TodayBlackoutBackground, TodayBlackoutForeground, TodayHoverBackground, TodayPressedBackground, TodaySelectedInnerBorderBrush.
+
+### Showcasing the TodayBackground property in the month, year and decade views
+![Default Today Background.](images/DefaultTodayBackground.png)
+![Default Today Month Background.](images/TodayMonthBackground.png)
+![Default Today Year Background.](images/TodayYearBackground.png)
+
+### Showcasing the TodayHoverBackground property - current date is hovered
+![Hover Today Background.](images/HoverTodayBackground.png)
+
+### Showcasing the TodayPressedBackground property - current date is pressed
+![Pressed Today Background.](images/PressedTodayBackground.png)
+
+### Showcasing the TodayBlackoutBackground and TodayBlackoutForeground properties - the old property CalendarView.IsTodayBlackedOut is True
+![Blacked Out Today Background And Foreground.](images/BlackoutTodayBackgroundAndForeground.png)
+
+### Showcasing the TodaySelectedInnerBorderBrush property - current date is selected
+![Selected Today Border.](images/SelectedTodayBorder.png)
+
+## Blacked Out styling
+The following properties of type Windows.UI.Xaml.Media.Brush are used to customize the calendar items that are blacked out in the month view:
+TodayBlackoutBackground, TodayBlackoutForeground, BlackoutStrikethroughBrush.
+
+### Showcasing the BlackoutStrikethroughBrush property - all Sundays are blacked out
+![Blacked Out Sundays.](images/BlackoutStrikethroughBrush.png)
+
+## Out Of Scope styling
+The following properties of type Windows.UI.Xaml.Media.Brush are used to customize the calendar items that are outside the current scope (month, year, or decade):
+OutOfScopeHoverForeground, OutOfScopePressedForeground.
+
+### Showcasing the OutOfScopeHoverForeground property - 4th of May is hovered
```

Comment:
Is this really "Hover" or "Over"? It's Hover if you have to hold the mouse steady for a moment to see the effect. If it happens right away then it's Over. (Same for all the rest of the 'Hover' properties.)

Reply:
It's really Over. I used Hover because the CalendarView control has been using Hover in existing properties: SelectedHoverBorderBrush, HoverBorderBrush.

Is intra-control consistency the least important criteria?